packages/sdk/src/SshConfigUtils.ts

```diff
@@ -71,7 +71,9 @@ export class SshConfigUtils {
                                     session.user = value;
                                     break;
                                 case "identityfile":
-                                    session.privateKey = value;
+                                    session.privateKey = path.normalize(
+                                        value.startsWith("~") ? path.join(os.homedir(), value.slice(2)) : value,
```

Comment:
Consider saving the result of `os.homedir()` to a variable before using it to avoid the extra function call